docs/reference/changelog.rst

```diff
@@ -68,6 +68,66 @@ Changelog
 
   For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
 
+8.4.1 (2024-Sep-20)
+-------------------
+
+Core
+====
+
+* Fix a regression where numeric entries in a ``snapcraft.yaml`` could not be
+  parsed.
+
+Bases
+#####
+
+core24
+""""""
+
+* Fix a regression where ``build-for`` could not be omitted in a ``platforms``
+  entry in a ``snapcraft.yaml`` file.
+
+* Fix a regression where ``--shell`` and ``--shell-after`` were not supported
+  for the ``pack`` command (`#4963`_).
+
+* Fix a regression where ``--debug`` would not shell into the build environment
+  if the packing step fails (`#4959`_).
+
+Plugins
+#######
+
+NPM
+"""
+
+* Fix a bug where NPM parts fail to build if the ``pull`` and ``build`` steps
+  did not occur in the same execution of Snapcraft.
```

Comment:
> execution

Do we mean "instance"? I'm not a fan of "execution" as a correlate of "executable".

Reply:
Yeah, I think "instance" is a better choice.